lib/widgets/compose_box.dart

```diff
@@ -167,6 +167,32 @@ class ComposeTopicController extends ComposeController<TopicValidationError> {
   /// that certain strings are not empty but also indicate the absence of a topic.
   bool get _isTopicVacuous => textNormalized == kNoTopicTopic;
 
+  /// The send destination as a string.
```

Comment:
```
compose: Change content input hint text if topic is empty and mandatory

Previously, "Message #stream > (no topic)" would appear as the hint text
when the topic input is empty but mandatory.

[…]
```

The commit message says what the hint text used to be; how about also saying what it is now?